### DISCUSSION TITLE
Definition of results from Simulations

### DISCUSSION THREAD

**Discussion by michefabb** [simnibs/simnibs#270] [9. Results and analysis]
Dear SimNIBS Experts,

I have used charm (v410a1) with the Freesurfer fsaverage5 midthickness cortical surfaces, then I ran some simulations; in the GUI, checking the corresponding box "Interpolate to cortical surface", I got the subject_overlays folder with the results in the vertices of the fsaverage5 midsurface that I passed during charm.

I would have some question about the results, since I was not able to find more information in the documentation:
- Enorm array, I assume this is the component of the vector perpendicular to the midthickness cortical surface
- Etang array, I guess this is the remaining part of the vector, perpendicular to Enorm and tangent to the midthickness cortical surface
- Emagn array, this is the magnitude of the E array, corresponding to Emagn=(Enorm^2+Etang^2)^0.5
- Eangl array, I was not able to figure out to what this refers...I would assume this is the angle in radians [rad] of the Etang component respect to a reference axis...but I was not able to understand respect to which reference...would you kindly clarify?
- D array, I could not find out what this array is (for TMS only)...could you kindly clarify?
- v array, I assume this is the additive membrane voltage induced by tDCs/TMS to the pyramidal cells membrane voltage, but I was not able to understand how this is computed: checking the references I found the literature for the lambdaE models (v is proportional to the Enorm vector with the membrane spatial constant lambda being the proportionality constant... v=lambda*Enorm) however checking the results it does not look this is the case (unless lambda changes value from point to point in SimNIBS...in this case how is it computed?) ...would you kindly clarify?

Thanks again for your usual great support!!

Mic

**Comment by jduemose** [comment]
Hi Mic

- You are right about Enorm, Etang, Emagn. There is a quick explanation [here](https://simnibs.github.io/simnibs/build/html/faq.html#electric-field-magnitude-normal).
- "Eangl" is the angle between the field vector (E) and the surface normal.
- `D` is dA/dt from the TMS coil (which induces the 'primary' E field - the E field as it would be in an infinite, homogeneous medium).
- `v` is the potential from which the secondary E field is calculated (as the gradient). The final E field is calculated from the primary and secondary fields. See for example https://link.springer.com/chapter/10.1007/978-3-030-43395-6_7 for further info

I agree with you that the documentation is somewhat lacking in this respect! Hope this helps.

Best,
Jesper

**Reply by michefabb** [reply]
@jduemose thank you for your quick answer!
Unfortunately I am unable to access the reference provided above, hopefully it is similar to what described in the below papers from Saturnino and Thielscher?
[https://iopscience.iop.org/article/10.1088/1741-2552/ab41ba/meta?casa_token=Ghfcok11I1QAAAAA:eCGfS28KRzTIsjyxc0V2aUex8IdNoPYrMIGwB53BqLacKUba_qYWZI0FwTHFgBlS4mdaFWIlJT4VAjktx7x6OyYt-nR3](url)

[https://link.springer.com/chapter/10.1007/978-3-030-21293-3_1](url)

[https://www.sciencedirect.com/science/article/pii/S1053811910010347?via%3Dihub](url)

**Reply by bjornsig** [reply]
yes
